app/src/main/java/app/revanced/integrations/youtube/utils/VideoUtils.java

```diff
@@ -130,6 +133,27 @@ public static void launchExternalDownloader(@NonNull String videoId) {
         }
     }
 
+    public static void launchPlaylistExternalDownloader(@NonNull String playlistId) {
+        try {
+            // use default downloader (YTDLnis) if user tries to download a playlist
+            // because most downloaders don't support playlist download
+            String downloaderPackageName = externalDownloaderPackageName.defaultValue;
+
+            final String content = String.format("https://www.youtube.com/playlist?list=%s", playlistId);
+
+            if (!checkPackageIsEnabled()) {
```

Comment:
`checkPackageIsEnabled` this ones should be changed

Reply:
why?

Reply:
> why?

this function check if the setted downloader is installed, not checking YTDLnis